parsons/bluelink/person.py

```diff
@@ -0,0 +1,242 @@
+import logging
+import json
+
+logger = logging.getLogger(__name__)
+
+
+class Person(object):
+    """
+    Instantiate Person Class.
+    Used for to upserting via Bluelink connector.
+    See: https://bluelinkdata.github.io/docs/BluelinkApiGuide#person-object
+
+    `Args:`
```

Comment:
Minor, but there are a few places the order of args here doesn't match with the order in the function